topic-controller/resources/openshift-template.yaml

```diff
@@ -16,14 +16,15 @@ rules:
 - apiGroups:
   - ""
   resources:
-  - configmaps
+  - ["configmaps", "events"]
   verbs:
   - get
   - list
   - watch
   - create
   - patch
   - update
+  - delete
```

Comment:
I was wondering if you really need all these verbs for `events` or just the `create` one ...

Reply:
Excellent point